requirements.txt

```diff
@@ -3,7 +3,7 @@ filetype==1.0.7
 geoalchemy2==0.8.5 # rq.filter: <0.9
 pyramid==1.10.8 # rq.filter: <2
 pyramid-debugtoolbar==4.9
-shapely==1.6.4.post1  # rq.filter: ==1.6.4.post1
+shapely==1.8.1  # rq.filter: == 1.8.1
```

Comment:
If we are no longer stuck with a specific version of shapely, you can simply remove the "rq.filter" part.